tests/Tests.Core/Xunit/ElasticsearchXunitRunOptions.cs

```diff
@@ -29,7 +31,16 @@ public ElasticsearchXunitRunOptions()
 			Generators.Initialize();
 		}
 
-		public override void OnBeforeTestsRun() => TestConfiguration.Instance.DumpConfiguration();
+		public override void OnBeforeTestsRun()
+		{
+			VerifierSettings.DerivePathInfo(
```

Comment:
`[ModuleInitializer]` is not available prior to NET5.

This snippet is moved from there.